code-cademy.html

```diff
@@ -1,9 +1,239 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
+	<meta charset="utf-8">
 	<title>codecademy</title>
+	<meta name="viewport" content="width=device-width">
+	<link rel="stylesheet" type="text/css" href="code-cademy.css">
+	<link href="https://use.fontawesome.com/releases/v5.0.7/css/all.css" rel="stylesheet">
 </head>
 <body>
 
+	<header>
+		<nav class="site-nav">
+			<ul>
+				<li><a href="">Catalog</a></li>
+				<li><a href="">Learn From Home</a></li>
+				<li><a href="">Pricing</a></li>
+				<li><a href="">For Business</a></li>
+
+			<ul class="nav">
+				<li><a href=""><b>Log in</b></a></li>
+				<li><a href=""><strong>Sign up</strong></a></li>
+			</ul>
+
+			</ul>
+		</nav>
+			<div class="fix"></div>
+	</header>
+
+	<div class="images">
+		<img src="images/code3.svg" style="margin-left: 88px; margin-bottom: -30px">
+		<img src="images/code2.svg" style="margin-bottom: -193px; position: relative; margin-left: 27px; z-index: 11">
+		<img src="images/code1.svg" style="margin-left: 377px;">
+		<img src="images/code1.webp" style="margin-top: -181px; margin-left: 113px; position: relative;">
+		<img src="images/code6.svg" style="margin-top: -263px; width: 43.4%; margin-left: 88px; position: relative;">
+		<div class="box">
+			<p>Join the Millions Learning to Code with Codecademy</p>
+		</div>
+	</div>
+
+	<div class="content1">
+		<h3>Get Started For Free</h3>
+		<div class="label1">
+		<label>Email</label>
+		<input type="text" name="email">
```

Comment:
this two lines are elements of a parent `div` so indent them properly to show that.
```
<div class="label1">
       <label>Email</label>
       <input type="text" name="email">
</div>

```

Apply this rule through your codebase